Corzbank.Data.Entities/Models/UserModel.cs

```diff
@@ -1,10 +1,28 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 using System.Text;
 
-namespace Corzbank.Data.Entities
+namespace Corzbank.Data.Entities.Models
 {
     public class UserModel
     {
+        [Required(ErrorMessage ="Firstname is required")]
+        public string Firstname { get; set; }
+
+        [Required(ErrorMessage = "Lastname is required")]
+        public string Lastname { get; set; }
+
+        [Required(ErrorMessage = "PhoneNumber is required")]
+        public string PhoneNumber { get; set; }
+
+        [Required(ErrorMessage = "Email is required")]
+        public string Email { get; set; }
```

Comment:
email should have a strict format, add validation for that

Reply:
resolved

Reply:
where?

Reply:
why did you delete all the annotations?
validation for email isn't present
required fields are not present